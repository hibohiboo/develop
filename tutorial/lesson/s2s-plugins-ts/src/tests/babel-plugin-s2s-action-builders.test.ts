import * as path from 'path'
import * as pluginTester from 'babel-plugin-tester'
import * as plugin from 'babel-plugin-s2s-action-builders'

const getFixturesPath = x => path.resolve(__dirname, '__fixtures__', x);

const fromWithStatePath = getFixturesPath('with-initial-state.js')

const testCases = [
  {
    title: 'basic',
    code: `
import { createAction } from 'redux-actions'
getCoinRequest
`,
  },
]

pluginTester({
  title: 'default',
  plugin,
  snapshot: true,
  tests: testCases,
})

