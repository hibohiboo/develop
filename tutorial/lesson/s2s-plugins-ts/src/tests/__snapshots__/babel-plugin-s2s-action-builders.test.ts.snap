// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default basic: basic 1`] = `
"
import { createAction } from 'redux-actions'
getCoinRequest

      ↓ ↓ ↓ ↓ ↓ ↓

import { createAction } from 'redux-actions';
let getCoinRequest;
let getCoinSuccess;
let getCoinFailure;
"
`;
